docs/faq.md

```diff
@@ -1,123 +1,150 @@
-FAQ
-==========
-
-版本相关
-""""""""""""""""
-
-:问:
-  FISCO BCOS v2.0与之前版本有哪些变化？
-:答:
-  请 :doc:`参考这里 <whats_new>`。
-
-:问:
-  FISCO BCOS v2.0都有哪些API？开发者如何与FISCO BCOS平台交互？
-:答:
-  1. FISCO BCOS v2.0提供jsonrpc接口，具体请 :doc:`参考这里 <api>`。
-  2. FISCO BCOS v2.0提供Java SDK帮助开发者快速实现应用，具体请 :doc:`参考这里 <sdk/index>`。
-  3. FISCO BCOS v2.0提供控制台帮助用户快速了解使用FISCO BCOS，具体请 :doc:`参考这里 <manual/console>`。
-
-:问:
-  FISCO BCOS v2.0如何搭建？
-:答:
-  - 建链脚本：使用简单，适合第一次接触FISCO BCOS的同学，具体请 :doc:`参考这里 <manual/build_chain>`。
-  - FISCO-Generator：更灵活，适合熟悉FISCO BCOS的用户和企业用户，具体请 :doc:`参考这里 <enterprise/index>`。
-
-:问:
-  FISCO BCOS v2.0的智能合约与之前版本合约有什么不同，兼容性如何？
-:答:
-  FISCO BCOS v2.0支持最新的Solidity合约，同时增加了precompare合约，具体请 :doc:`参考这里 <developer/index>`。
-
-:问:
-  国密和普通版本的区别有哪些？
-:答:
-  编译版本，证书，落盘加密，solidity编译java，web3sdk使用，具体请 :doc:`参考这里 <manual/guomi>`。
-
-:问:
-  是否支持从1.3或1.5升级到2.0版本？  
-:答:
+# FAQ
+
+## 版本相关
+
+问:
+  FISCO BCOS v2.0与之前版本有哪些变化？<br>
+答:
+  请 [参考这里](./whats_new)。
+
+问:
+  FISCO BCOS v2.0都有哪些API？开发者如何与FISCO BCOS平台交互？<br>
+答:
+  FISCO BCOS的API为
+  1. FISCO BCOS v2.0提供jsonrpc接口，具体请 [参考这里](./api)。
+  2. FISCO BCOS v2.0提供Java SDK帮助开发者快速实现应用，具体请 [参考这里](./sdk/index.html)。
+  3. FISCO BCOS v2.0提供控制台帮助用户快速了解使用FISCO BCOS，具体请 [参考这里](./manual/console.md)。
+
+问:
+  FISCO BCOS v2.0如何搭建？<br>
+答:
+  FISCO BCOS支持多种搭建方式，常用方式有：
+  - 建链脚本：使用简单，适合第一次接触FISCO BCOS的同学，具体请 [参考这里](./manual/build_chain.md)。
+  - FISCO-Generator：更灵活，适合熟悉FISCO BCOS的用户和企业用户，具体请 [参考这里](./enterprise/index.html)。
+
+问:
+  FISCO BCOS v2.0的智能合约与之前版本合约有什么不同，兼容性如何？<br>
+答:
+  FISCO BCOS v2.0支持最新的Solidity合约，同时增加了precompare合约，具体请 [参考这里](developer/index.html)。
+
+问:
+  国密和普通版本的区别有哪些？<br>
+答:
+  编译版本，证书，落盘加密，solidity编译java，web3sdk使用，具体请 [参考这里](./manual/guomi.md)。
```

Comment:
Description is a little simple. Maybe need link to a more detailed reference.

Reply:
done